rust/src/milla/api.rs

```diff
@@ -337,6 +359,12 @@ pub(crate) fn internal_set_tile(
     if let Some(value) = agent_b {
         tile.gases.set_agent_b(value);
     }
+	if let Some(value) = hydrogen {
+		tile.gases.set_hydrogen(value);
+	}
+	if let Some(value) = water_vapor {
+		tile.gases.set_water_vapor(value);
+	}
     // Done sooner because we need innate heat capacity to calculate thermal energy from
```

Comment:
Your intentation for the rust code seems off, it looks like you used tabs instead of spaces. Please update to spaces to be consistent